external/os-autoinst-common/cpanfile

```diff
@@ -17,7 +17,7 @@ on 'develop' => sub {
     requires 'Code::TidyAll';
     requires 'Perl::Critic';
     requires 'Perl::Critic::Community';
-    requires 'Perl::Tidy', '== 20230912';
+    requires 'Perl::Tidy', '== 20240202';
```

Comment:
Before we pull in that I would like us to sort us the conflict that was created that os-autoinst-common relies on the most recent releases on CPAN whereas in other products we rely on the version in Tumbleweed/Leap

Reply:
Yeah. I created https://github.com/os-autoinst/os-autoinst-common/pull/45 and that passes now, so we're back to version 20230912 after that is merged.